### DISCUSSION TITLE
Testing Idena node: release candidate 0.26.7-rc1

### DISCUSSION THREAD

**Discussion by midenaio** [idena-network/idena-go#762] [General]
We invite you to test the release candidate version of the Idena node: 0.26.7-rc1.
Please [download](https://github.com/idena-network/idena-go/releases/tag/v0.26.7-rc1) it manually if you'd like to join the testing.

Changes:

- Performance improvements
- Updating IPFS version
- Bug fixes

We plan to release these changes in two days unless negative feedback is received

**Comment by Fragilant** [comment]
The rc1 update runs smooth on two of the smallest (1CPU/2GB) hetzner vps. Thanks for your passion :-)
Is there any doc regarding the ipfs-folder / optimization? The directory size hits 16GB an me (and maybe many others) have to upgrade their vps or at least their hdds.

**Reply by midenaio** [reply]
@Fragilant Thanks for your feedback

You can run Idena node with a config file (see [example](https://docs.idena.io/docs/developer/node/general/#json-config)) specifying the share of blocks/flips stored in the local IPFS folder:
`BlockPinThreshold`: by default 30% of blocks are stored 
`FlipPinThreshold`: by default 50% of new flips are stored

**Reply by Fragilant** [reply]
Thanks for your reply. I did not read the conf file closely enough :|
I resized the hdds - but next time i´ll try to set the tresholds lower. Anyway: ty :-)
